src/chapters/angular-wk1/studio.rst

```diff
@@ -0,0 +1,61 @@
+Studio: Angular, Part 1
+========================
+
+Angular allows us to build webpages in small pieces, which keeps us organized, and allows us to focus on one thing at a time.
+
+Throughout our work for this class, we have learned about the Angular file structure, templates, and components. Let's put these concepts into practice in our studio.
+
+Throughout the next three classes we will be building a Mission Planning Dashboard to practice our Angular skills.
+
+Mission Planning Dashboard
+--------------------------
+
+A useful, and common front end application is a dashboard. It shows quick information about a topic, so individuals using the web app can be well informed and make good decisions.
+
+We will be making a space Mission Planning Dashboard.
+
+This Angular project needs to include five sections:
+    #. A header that displays the mission, and rocket name
+    #. A list that displays the names of the crew
+    #. A list that dipslays crew member roles
+    #. A list that displays required equipment
+    #. A list that displays the countries reprStudio: Angular, Part 1
+
+Angular allows us to build webpages in small pieces, which keeps us organized, and allows us to focus on one thing at a time.
+
+Throughout our work for this class, we have learned about the Angular file structure, templates, and components. Let's put these concepts into practice in our studio.
+
+Throughout the next three classes we will be building a Mission Planning Dashboard to practice our Angular skills.
+
+Mission Planning Dashboard
+--------------------------
+
+A useful, and common front end application is a dashboard. It shows quick information about a topic, so individuals using the web app can be well informed and make good decisions.
+
+We will be making a space Mission Planning Dashboard.
+
+This Angular project needs to include five sections:
+    #. A header that displays the mission, and rocket name
+    #. A list that displays the names of the crew
+    #. A list that displays crew member roles
+    #. A list that displays required equipment
+    #. A list that displays the countries represented by the crew
+
+Each section needs to be it's own component, and they all need to be organized by the app component.
+
+In future lessons we will learn how to expand this project by using more Angular tools that will make the application more interactive. For the purposes of today's studio our web application will not be interactive, and will simply display static information.
+
+Project Data
+------------
+
+Data you will need for this project:
+    - Mission name: Apollo 11
+    - Rocket name: Saturn V
+    - Crew member names & roles: Commander Neil Armstrong, Command Module Pilot Michael Collins, Lunar Module Pilot Buzz Aldrin
+    - Equipment: Command Module, Lunar Module, 3 flight suits, 2 moon suits, 30 meals, 100 litres of water
+    - Countries: United States
+
+Bonus Mission
+-------------
+
+TODO: Info about CSS here?
```

Comment:
Making the site look pretty is a good way to encourage continued attention and work with the template.